examples/sqlfluff-plugin-example/setup.py

```diff
@@ -0,0 +1,9 @@
+"""Setup file for example plugin."""
+from setuptools import setup
+
+setup(
+    name="sqlfluff-plugin-example",
+    install_requires="sqlfluff",
```

Comment:
Should we consider specifying a minimum version of sqlfluff here?

Reply:
optional minimum and maxium sqlfluff versions would be good here. I don't think they should be *required*, but as an example file it should probably demonstrate the functionality.

Reply:
The reason for specifying a minimum could be if the rules in the plugin rely on certain APIs or behaviors being present in the core. So plugins would not be required to use them, but in practice, I think they will often want to.

Reply:
yes! my bad, actually the minimum will have to be "the release that implements pluggy" at the very least